tests/log_file/build_log_file_test.py

```diff
@@ -0,0 +1,41 @@
+import logging
+import os
+import pytest
+
+from twister2.log_file.build_log_file import BuildLogFile
+
+logger = logging.getLogger(__name__)
+
+
+@pytest.fixture
+def build_log(resources) -> BuildLogFile:
```

Comment:
I would use tmp_dir instead resources to not create temporary files in the repository
https://docs.pytest.org/en/6.2.x/tmpdir.html

Reply:
Done, thanks for tipp :)